examples/azureiot_native_networking/azureiot_central_commands.py

```diff
@@ -21,11 +21,13 @@
 print("Connecting to WiFi...")
 wifi.radio.connect(secrets["ssid"], secrets["password"])
 
+pool = adafruit_connection_manager.get_radio_socketpool(wifi.radio)
```

Comment:
These changes are common for on-board:
1. get `pool` and `ssl_context` via `adafruit_connection_manager`
2. Use pool for ntp
3. Pass to azure method